app/views/cookies/show.html.slim

```diff
@@ -0,0 +1,96 @@
+h1.govuk-heading-xl
+  | Cookies
+p.govuk-body
+  == t('.paragraph_1', service_name: link_to(t('.service_name'), root_path, class: 'govuk-link'))
+p.govuk-body
+  = t('.paragraph_2', service_name: t('.service_name'))
+p.govuk-body
+  == t('.paragraph_3', how_to_manage: link_to(t('.how_to_manage'),
+                                      "https://ico.org.uk/for-the-public/online/cookies",
+                                      class: 'govuk-link'))
+h2.govuk-heading-l class="govuk-!-margin-top-6"
+  = t('.essential_cookies')
+p.govuk-body
+  = t('.essential_cookie_explainer')
+table.govuk-table
+  caption.govuk-table__caption
+    = t('.essential_cookies_used')
+  thead.govuk-table__head
+    tr.govuk-table__row
+      th.govuk-table__header[scope="col"]
+        = t('.name')
+      th.govuk-table__header[scope="col"]
+        = t('.purpose')
+      th.govuk-table__header[scope="col"]
+        = t('.expires')
+  tbody.govuk-table__body
+    tr.govuk-table__row
+      th.govuk-table__header[scope="row"]
+        | optional_cookie_choice
+      td.govuk-table__cell
+        = t('.saves_settings')
+      td.govuk-table__cell
+        = t('.one_year')
+    tr.govuk-table__row
+      th.govuk-table__header[scope="row"]
+        = request.env['rack.session.options'].instance_variable_get('@by').key
```

Comment:
Can we use something with a more descriptive name here? also in lines 78 and 85 the names are not very clear. Perhaps we change the _ga to GoogleAnalytics and rename this one to session_cookie?